### ISSUE TITLE
decreased circulating glutathione peroxidase activity

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#4230]
Preferred term label (e.g., asplenia)

1. abnormal circulating glutathione peroxidase activity
2. decreased circulating glutathione peroxidase activity
3. increased circulating glutathione peroxidase activity

Synonyms (e.g., absent spleen)

1. abnormal circulating GSH-Px activity; abnormal circulating GSHPx activity; abnormal circulating GPX activity
2. reduced circulating glutathione peroxidase activity; reduced circulating GSH-Px activity; decreased circulating GSH-Px activity; decreased circulating GSHPx activity; decreased circulating GPX activity
reduced circulating GSHPx activity; reduced circulating GPX activity
2. increased circulating GSH-Px activity; increased circulating GSHPx activity; increased circulating GPX activity

Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal glutathione peroxidase activity (MP:0011601)
2. abnormal circulating glutathione peroxidase activity (new term)
3. abnormal circulating glutathione peroxidase activity (new term)

Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. anomaly in the ability to catalyze the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O in the circulation
2. reduction in the ability to catalyze the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O in the circulation
3. increase in the ability to catalyze the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O in the circulation

PMID:27871290

If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5256-8683

**Comment by anna-anagnostop** [comment]
NOTE: the decreased/increased, but NOT the abnormal circulating glutathione peroxidase activity parent term have already been requested via #4214 (still open).
